sci-mathematics/cgal/cgal-4.11.recipe

```diff
@@ -96,18 +100,27 @@ BUILD_REQUIRES="
 	"
 BUILD_PREREQUIRES="
 	cmd:cmake
-	cmd:make
 	cmd:gcc$secondaryArchSuffix
+	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage uriparser$secondaryArchSuffix \
+	"$libDir"/libCGAL.so.$libVersion \
+	"$libDir"/libCGAL_Core.so.$libVersion \
+	"$libDir"/libCGAL_ImageIO.so.$libVersion \
+	"$libDir"/libCGAL_Qt5.so.$libVersion
+
 BUILD()
 {
 	mkdir -p build && cd build
-	cmake .. -DCMAKE_INSTALL_PREFIX=$prefix \
+	cmake .. $cmakeDirArgs \
 		-DCMAKE_BUILD_TYPE=Release \
 		-DWITH_Eigen3=ON \
-		-DCGAL_INSTALL_CMAKE_DIR="lib/cmake"
+		-DBUILD_TESTING=ON \
+		-DCGAL_INSTALL_CMAKE_DIR="$relativeLibDir/cmake" \
```

Comment:
A bit of tweaks to [install directories][0]

```diff
+		-DCGAL_INSTALL_DOC_DIR="$docDir" \
+		-DCGAL_INSTALL_MAN_DIR="$manDir/man1"
```

**Note:** If the installed docs are in HTML format, please put them into `developDocDir`

> `$developDocDir`: The absolute base directory for development-related documentation specific to the port being built. Documentation in html- or pdf-format should be put here, manpages should be put into `$manDir`

[0]: https://github.com/CGAL/cgal/blob/releases/CGAL-4.11/Installation/CMakeLists.txt#L778